packages/expo-font/android/src/main/java/expo/modules/font/FontLoaderModule.kt

```diff
@@ -22,9 +22,17 @@ open class FontLoaderModule : Module() {
     get() = appContext.reactContext ?: throw Exceptions.ReactContextLost()
 
   override fun definition() = ModuleDefinition {
+    val customNativeFonts = queryCustomNativeFonts()
+    // could be a Set, but to be able to pass to JS we keep it as an array
+    var loadedFonts: List<String> = customNativeFonts
+
     Name("ExpoFontLoader")
 
-    Constants("customNativeFonts" to queryCustomNativeFonts())
+    Constants("customNativeFonts" to customNativeFonts)
+
+    Property("loadedFonts") {
```

Comment:
what's the expected usage pattern of this look like? From my reading, it'd look something like:

```typescript
let loadedFonts = ExpoFont.loadedFonts;
await ExpoFont.loadAsync(...);
loadedFonts = ExpoFont.loadedFonts; // this is now different and the local state must be updated
```

I wonder if it might make more sense to make `loadedFonts` a method (`getLoadedFonts` to better indicate that it is not constant and probably needs to be queried each time it is used? This suggestion is optional, I don't feel too strongly, just want to see what you think.

Reply:
Yes, that's the expected usage. This reminds me I didn't expose this api in JS at all 😀 . Will do that. I'll see if it makes more sense and a function or property.

Reply:
added `getLoadedFonts()` function on the JS side